docs/examples/fixed-nrel.py

```diff
@@ -0,0 +1,55 @@
+"""
+Flag Sunny Days for a Fixed-Tilt System
+=======================================
+
+Flag sunny days for a fixed-tilt PV system.
+"""
+
+# %%
+# Identifying and masking sunny days for a fixed-tilt system is important
+# when performing future analyses that require filtered clearsky data.
+# For this example we will use data from the fixed-tilt NREL SERF East system
+# located on the NREL campus in Colorado, USA, and generate a sunny day mask.
+# This data set is publicly available via the PVDAQ database in the
+# DOE Open Energy Data Initiative (OEDI)
+# (https://data.openei.org/submissions/4568). This data is timezone-localized.
```

Comment:
Mention the relevant OEDI system_id

Reply:
Updated in commit https://github.com/pvlib/pvanalytics/pull/148/commits/0c8c07e829f219dcc2de10a0b1d9b5eb8f8af930